cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs

```diff
@@ -118,19 +118,6 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	state_version: 1,
 };
 
-#[cfg(not(feature = "state-trie-version-1"))]
```

Comment:
I am not sure if it is still needed, but I wouldn't say it is a duplicate, they have different `state_version`:
`state_version: 1,` for `#[cfg(feature = "state-trie-version-1")]`
`state_version: 0,` for `#[cfg(not(feature = "state-trie-version-1"))]`